src/main/resources/assets/js/application/recruiter/services/personsService.js

```diff
@@ -0,0 +1,36 @@
+define(['application/recruiter/tdprRecruiterModule'], function (tdprRecruiterModule) {
+    tdprRecruiterModule.service('personsService', function ($http, dateFilter) {
```

Comment:
we add prefixes to all files to keep consistency within a project. This should be named tdprPersonsService